### ISSUE TITLE
German language testing

### ISSUE THREAD

**Issue by davidjharder** [getsolus/solus-mate-transition-tool#5]
Hey Joey, I am testing the tool in regard of german translation (de_DE). What would I need to do to let transition fail on purpose to check error messages? I can think of not giving enough disk space and cutting internet-connection during download. I did the latter while it was downloading Budgie. No error messages here. After a short while, it just went from `Downloading...` to `Finished` and stopped there. Is this desired behaviour? After restarting tool, I could restart the download and tool would then install Budgie.
Then I have reset VM and started from beginning.
After successful installation of Budgie or XFCE, tool again just says  `Finished`. (Just to be clear: translation was fine. Tool originally printed `Beendet` here and before. I refer here to original english strings for comprehension.)
I had to figure out myself that it's time to close the window and reboot.
I have never seen this dialog `{DE} Successfully installed, please reboot to continue.`
After reboot, if you then login to MATE anyways, transition-tool autostarts.
It does not autostart however for removal of MATE if you login to Budgie/XFCE. Here you have to start it manually to proceed.
Also, after installation of XFCE and removal of MATE, this dialog appears: `Successfully Completed Transition to {}` `We hope you enjoy using your new desktop environment.`
This dialog however does not appear after installation of Budgie and removal of MATE. Here, it just says `Finished` and you have again to figure out yourself to just close the window.

If you rerun transition tool after removal of MATE you get an error dialog that says: `Logout and login to the MATE session first to continue` This string is actually in english. That is the only time I have seen a non-translated string.
All german translations also have been correct.

Apart of my above comments, tool works great.

_Originally posted by @sebo505 in https://github.com/getsolus/packages/issues/1134#issuecomment-2087218969_

**Comment by joebonrichie** [comment]
I've split the issues found here into #6 and #7. 

> If you rerun transition tool after removal of MATE you get an error dialog that says: Logout and login to the MATE session first to continue This string is actually in english. That is the only time I have seen a non-translated string.
All german translations also have been correct.

This may be due to this: https://github.com/getsolus/solus-mate-transition-tool/commit/35d456230bde8f465080c6ec005588fc60dbde31. I imagine the existing translations may need to be updated.

**Comment by davidjharder** [comment]
Yeah, I'll need to do a translation update to get that straggler sorted

**Comment by sebo505** [comment]
> This may be due to this: [35d4562](https://github.com/getsolus/solus-mate-transition-tool/commit/35d456230bde8f465080c6ec005588fc60dbde31). I imagine the existing translations may need to be updated.

I see here two issues: There is no `Logout and login to the MATE session first to continue` string on transiflex.
And: shouldn't tool know that it itself removed MATE, hence, there is no MATE session to login anymore

**Comment by joebonrichie** [comment]
> And: shouldn't tool know that it itself removed MATE, hence, there is no MATE session to login anymore

This is mostly for if you run a tool on a different desktop environment first e.g. a random user tries to run the tool on their plasma or gnome install.
